src/store/transaction.js

```diff
@@ -35,10 +35,118 @@ const managers = [
     pageInfo: {
       pageSize: 20
     },
-    filter: {
-      0: 'All',
-      ...Constants.TransactionType
-    }
+    filter: [
```

Comment:
can we move this filter under config?
because we can reuse for detail-pages's card filter.

Reply:
Yes, good idea about config file. I will see what we can do